components/xsd-fu/xsd-fu

```diff
@@ -190,26 +192,6 @@ def main(model, opts):
             content = content.render(encoding='utf-8')
             customAsXMLElementPropertyContent[propertyName] = content
         # END Custom templates for asXMLElment()
-        parents = model.resolve_parents(obj.name)
```

Comment:
It's not removed; it's moved above so that the resolved parents are accessible in the ancillary template fragments.